docs/admin/provisioners.md

```diff
@@ -10,22 +10,23 @@ By default, the Coder server runs [built-in provisioner daemons](../cli/server.m
 
 - **Reduce server load**: External provisioners reduce load and build queue times from the Coder server. See [Scaling Coder](./scale.md#concurrent-workspace-builds) for more details.
 
-> External provisioners are in an [alpha state](../contributing/feature-stages.md#alpha-features) and the behavior is subject to change. Use [GitHub issues](https://github.com/coder/coder) to leave feedback.
-
-## Running external provisioners
-
 Each provisioner can run a single [concurrent workspace build](./scale.md#concurrent-workspace-builds). For example, running 30 provisioner containers will allow 30 users to start workspaces at the same time.
 
 Provisioners are started with the [coder provisionerd start](../cli/provisionerd_start.md) command.
 
-### Authentication
+## Authentication
+
+The provisioner daemon must authenticate with your Coder deployment.
 
-The provisioner server must authenticate with your Coder deployment. There are two authentication methods:
+Set a [provisioner daemon pre-shared key (PSK)](../cli/server.md#--provisioner-daemon-psk) on the Coder server and start the provisioner with
+`coder provisionerd start --psk <your-psk>`. If you are [installing with Helm](../install/kubernetes#install-coder-with-helm),
+see the [Helm example](#example-running-an-external-provisioner-with-helm) below.
 
-- PSK: Set a [provisioner daemon PSK](../cli/server#--provisioner-daemon-psk) on the Coder server and start the provisioner with `coder provisionerd start --psk <your-psk>`
-- User token: [Authenticate](../cli.md#--token) the Coder CLI as a user with the Template Admin or Owner role.
+> Coder still supports authenticating the provisioner daemon with a [token](../cli.md#--token) from a user with the
+> Template Admin or Owner role. This method is deprecated in favor of the PSK, which only has permission to access
+> provisioner daemon APIs. We recommend migrating to the PSK as soon as practical.
```

Comment:
If we supported API token scopes, would you be open to us supporting both, or are there other reasons for deprecating user token auth?

Something I like about relying on user authentication is that this token could be rotated without restarting the Coder server and #7992 could be used down the road to limit specific user tokens to specific provisioners versus full API access. 

It does feel a little odd passing `CODER_SESSION_TOKEN` to the provisioner, especially if it is a long-lived token, but that could be said about any automation somebody is trying to do with Coder.

I know it also requires that you [create a token](https://coder.com/docs/v2/latest/cli/tokens_create) prior to starting a provisioner and you can only create a token on behalf of another user (machine account) via the REST API which is clunky.

Are there other drawbacks of the token auth I am missing?

Reply:
> We should speak with our large customers to get their take on how they'd like to do this auth.

Bet you a 🌮 someone is going to ask for Azure Workload Identity